### DISCUSSION TITLE
Locked Move to One Dimension

### DISCUSSION THREAD

**Discussion by Remodor** [PintaProject/Pinta#1180] [Q&A] [answered]
Hey, thanks for this awesome tool!

When moving a selection, I was intuitively using `shift` to lock a move to one dimension as in so many other apps, like Krita, LibreOffice, Paint 3D (not Paint though 😄), draw.io and probably a bunch I cannot remember.

I checked the features, Q&A, and documentation and was fairly surprised to not finding any mentioning of this. Maybe it is not used as much as I thought or I missed something..

My questions are:
- Is there such a feature in Pinta?
- Is there a feature request or should I add one?

**Comment by cameronwhite** [comment, marked as answer]
I don't think there is an existing feature request for this, so feel free to go ahead and submit one!
